cmd/lncli/cmd_build_route.go

```diff
@@ -88,7 +88,7 @@ func buildRoute(ctx *cli.Context) error {
 		return err
 	}
 
-	printJSON(route)
```

Comment:
can remove the `printJSON` method now?

Reply:
no there are still a few others that use it, mostly for things that encode outpoints